src/components/Login/accountConnection/index.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Button,
+  Flex,
+  Icon,
+  Link,
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  Text,
+} from "@chakra-ui/react";
+import React, { Dispatch, SetStateAction } from "react";
+import MainLogoSvg from "@/assets/svg/CourtCanva-main-LOGO.svg";
+import { FcGoogle } from "react-icons/fc";
+import ModalOperator from "@/components/Login/ModalOperater";
+import { api } from "@/utils/axios";
+import { GoogleLoginRes } from "@/components/Login/SelectLogin";
+import { updateUserInfo } from "@/store/reducer/userSlice";
+import { useStoreDispatch } from "@/store/hooks";
+
+interface Props {
+  existedUserInfo: GoogleLoginRes | null;
+  setStep: Dispatch<SetStateAction<number>>;
+  updateLoginData: (data: GoogleLoginRes) => void;
+  onClose: () => void;
+}
+
+const AccountConnection: React.FC<Props> = ({
+  existedUserInfo,
+  setStep,
+  updateLoginData,
+  onClose,
+}) => {
+  const dispatch = useStoreDispatch();
+  const onConnect = async () => {
+    const { data } = await api("/user/connect", {
```

Comment:
Is there possible to add interface to describe "data" variable？

Reply:
> Is there possible to add interface to describe "data" variable？

Yes, it is. I could handle it in the same way as in "SelectLogin.ts" to assign it to a variable like "googleLoginRes" to give it strong type :)